core/src/main/scala/scalaz/Adjunction.scala

```diff
@@ -117,7 +117,7 @@ sealed abstract class AdjunctionInstances {
     }
 
   implicit def writerReaderAdjunction[E]: Adjunction[Writer[E, ?], Reader[E, ?]] =
-    new Adjunction[Writer[E, ?], Reader[E, ?]] {
+    new Adjunction[Writer[E, ?], Reader[E, ?]]()(WriterT.writerTraverse[E], implicitly) {
```

Comment:
The real solution is to reorder the implicits in `WriterT`. I wonder why this didn't come up in #1632 (maybe something changed between 7.2.x and 7.3.x). I can help with this after scalacheck is out.

Reply:
fixed